bw-dev

```diff
@@ -246,6 +246,9 @@ case "$CMD" in
     remove_remote_user_preview_images)
         runweb python manage.py remove_remote_user_preview_images
         ;;
+    erase_deleted_user_data)
```

Comment:
This has a function but no command: `./bw-dev erase_deleted_data` shows an error.

Reply:
Huh it seems to work for me, what error are you getting?

Reply:
scrap that, I was a little hasty with my testing and turns out I entered the wrong command 🙃